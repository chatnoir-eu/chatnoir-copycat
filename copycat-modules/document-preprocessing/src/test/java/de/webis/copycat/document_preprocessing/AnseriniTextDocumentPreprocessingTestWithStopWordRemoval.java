package de.webis.copycat.document_preprocessing;

import org.approvaltests.Approvals;
import org.junit.Test;

public class AnseriniTextDocumentPreprocessingTestWithStopWordRemoval extends DocumentPreprocessingTest {

	public AnseriniTextDocumentPreprocessingTestWithStopWordRemoval() {
		super(new AnseriniDocumentTransformation("JsoupStringTransform"));
	}

	@Test
	public void approveWebisHtmlPage() {
		Approvals.verify(preprocessedHtml("webis-de"));
	}
	
	@Test
	public void approveToucheHtmlPage() {
		Approvals.verify(preprocessedHtml("touche-webis-de"));
	}
	
	@Test
	public void approveSigirHtmlPage() {
		Approvals.verify(preprocessedHtml("sigir-org-2021-resource"));		
	}
}
