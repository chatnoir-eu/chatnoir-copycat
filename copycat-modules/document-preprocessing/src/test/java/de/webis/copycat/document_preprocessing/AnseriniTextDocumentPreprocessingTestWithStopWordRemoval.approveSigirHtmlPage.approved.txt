call resource papers sigir 2021 skip content main menu home submitmenu toggle call full papers call short papers call resource papers call demonstrations call tutorials call workshop proposals call doctoral consortium papers call perspectives papers checklist strengthen ir paper call sirip 2021 industry track attendmenu toggle registration policy against harassment programmenu toggle program glance keynote speakers workshops doctoral consortium tutorials sirip awards social programs banquet photos organization sponsorsmenu toggle supporters sponsors call sigir 2021 sponsorships call resource papers annual sigir conference major international forum presentation new research results demonstration new systems techniques broad field information retrieval ir 44th acm sigir conference held online july 11 15 2021 welcomes contributions related any aspect information retrieval access including theories foundations algorithms evaluation analysis applications conference program chairs invite those working areas related ir submit high impact original papers review year we have special track resource papers separate from regular full short papers conference resource paper chairs invite those working areas related ir submit original resource papers which presented posters conference submissions peer reviewed accepted resource papers published conference proceedings important dates time zone anywhere earth aoe resource paper abstracts due tue feb 23 2021 resource papers due tue mar 2 2021 resource paper notifications wed apr 14 2021 what resource resources track seeks submissions from both academia industry describe resources available community process methodology building those resources lessons learned resources include restricted test collections information retrieval access tasks documentation designs protocols evaluation tasks e.g novel task designs implemented evaluation forums labelled datasets machine learning software tools services information retrieval access software tools services evaluating analyzing information retrieval access systems review criteria authors should aware resource paper track use set review criteria different from regular full short papers outlined below novelty what new about resource does resource represent incremental advance something more dramatic availability resource available reviewer time review discrepancies between what described what available licensing terms use sufficiently open allow most academic industry researchers access resource resource data collected from people do appropriate human subjects control board procedures appear have been followed utility resource well documented what level expertise do you expect required make use resource tutorials examples do resemble actual uses toy examples resource data appropriate tools provided loading data resource data provenance source preprocessing cleaning aggregation stages clearly documented predicted impact what ir research activity enabled availability resource does resource advance well established research area brand new one do you expect resource useful long time need curated updated latter planned how large anticipated research user community grow shrink next few years submission guidelines resource papers must describe original work has been previously published accepted publication elsewhere simultaneously submitted currently under review another journal conference submissions resource research papers must english pdf format most 6 pages including figures length unrestricted space references current acm two column conference format suitable latex word overleaf templates available from acm website use sigconf proceedings template submissions should submitted electronically via easychair resource paper track https easychair.org conferences conf sigir2021 least one author each accepted paper required register present work conference submission policy resource paper review process single blind which means reviewers aware names affiliations paper authors therefore unlike other paper tracks sigir 2021 need hide author information submission authors should carefully go through acm authorship policy before submitting paper submissions which violate preprint policy length formatting requirements plagiarized subject desk rejection chairs also permitted double submit content both resource track other track s sigir 2021 e.g resource paper building dataset full paper containing construction process dataset experiment section support identification reviewers conflicts interest full author list must specified submission time authors should note changes author list after submission deadline allowed without permission from pc chairs resource paper chairs yiqun liu tsinghua university p.r.c ian soboroff nist u.s.a contact further information please contact sigir 2021 resource paper co chairs email sigir2021 resource easychair dot org copyright Â© 2021 special interest group information retrieval scroll top